generator/src/test/java/com/scottlogic/deg/generator/utils/Pair.java

```diff
@@ -0,0 +1,45 @@
+package com.scottlogic.deg.generator.utils;
+
+import java.util.Objects;
+
+public class Pair<T1, T2> {
```

Comment:
I understand that this type is replacing one used from `javafx.util`, but can it be better named based on what is using it?

Reply:
Removed Pair class entirely, replaced with bespoke private class inside ProfileViolationTests. Also rewritten the test profile generation method to reduce cognitive complexity.